### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by agamm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ npm run test:macos
   - copy the last json content from the server folder to `incidentDailyReportTemplate.json`
   - while the project is running `(npm start)` the page will be auto-reloaded at `localhost:8082`
 
+To debug the automation/docker image behind sane pdf reports, you can use your own automation with the `reports.pdf.script` setting (use in the troubleshooting tab).
+
```
Comment (idanizi-panw) [inline README.md:93]: @agamm maybe worth adding:
and also change the docker image at the automation settings (show filter `deprecated` automations to see it)
image

**Review by idanizi-panw — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (agamm) [reply README.md:93]: Add that image?

Comment (idanizi-panw) [reply README.md:93]: yes please

Comment (idanizi-panw) [reply README.md:93]: @agamm any update?

**Pull request merged**